translations/nl.json

```diff
@@ -28,6 +28,11 @@
 	"UPDATE_NOTIFICATION_MODULE": "Update beschikbaar voor {MODULE_NAME} module.",
 	"UPDATE_INFO_SINGLE": "De huidige installatie loopt {COMMIT_COUNT} commit achter op de {BRANCH_NAME} branch.",
 	"UPDATE_INFO_MULTIPLE": "De huidige installatie loopt {COMMIT_COUNT} commits achter op de {BRANCH_NAME} branch.",
+<<<<<<< HEAD
```

Comment:
Please remove those GIT lines indicating where the merge conflict happened @Travelbacon